contracts/modules/LoanOpenings.sol

```diff
@@ -511,11 +518,12 @@ contract LoanOpenings is
      *     receiver: receiver of funds (address(0) assumes borrower address).
      *     manager: delegated manager of loan unless address(0).
      * @param sentValues The values to send:
-     *     newRate: New loan interest rate.
+     *     interestRate: New loan interest rate.
      *     newPrincipal: New loan size (borrowAmount + any borrowed interest).
-     *     torqueInterest: New amount of interest to escrow for Torque loan (determines initial loan length).
+     *     interestInitialAmount: New amount of interest to escrow for Torque loan (determines initial loan length).
      *     loanTokenReceived: Total loanToken deposit (amount not sent to borrower in the case of Torque loans).
-     *     collateralTokenReceived: Total collateralToken deposit.
+     *     collateralTokenSent: Total collateralToken deposit.
```

Comment:
if `collateralTokenReceived` will be added to the struct then natspec comments shoud be raplaced respectively
```
*     collateralTokenReceived: Total collateralToken deposit.
```